### PR TITLE
Added Documentation on setting device token through user choice input event for Customer IO

### PR DESCRIPTION
**Pull request by desusai7**
…e event for customerIO

## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/marketing/customer.io.mdx [hunk 1]
```diff
@@ -186,6 +186,37 @@ rudderClient!!.rudderContext.putDeviceToken(getDeviceToken())
     </TabPanels>
 </Tabs>
 
+Alternatively you can also enter an event name of your choice in the Customer IO Settings page as shown below:
+
+<img src="../../assets/customer.io_devicetoken_event.png" /><span class="imageTitle">Customer.io Connection Settings</span>
```
Comment (saikumarrs) [inline destinations/marketing/customer.io.mdx:191]: I didn't realize it earlier but the label name of the newly introduced field is pretty lengthy. Please try to come up with a shorter name for the field and you can move the existing verbose description to the footer note.

Comment (ameypv-rudder) [reply destinations/marketing/customer.io.mdx:191]: cc @desusai7 

Suggestion for the field: "Event name after setting device token". Thoughts, @saikumarrs?

Comment (desusai7) [reply destinations/marketing/customer.io.mdx:191]: Okay got it. 
The suggestion made by @ameypv-rudder looks good to me.
WDYT @saikumarrs ?

Comment (saikumarrs) [reply destinations/marketing/customer.io.mdx:191]: LGTM. 
Alternate suggestion,
Label: "Event fired after setting device token"
Footer note: "Name of the event which is fired immediately after setting the device token"

Comment (desusai7) [reply destinations/marketing/customer.io.mdx:191]: Okay, made changes accordingly

File: destinations/marketing/customer.io.mdx [hunk 2]
```diff
@@ -26,47 +26,69 @@ Before configuring your source and destination on the RudderStack app, please ch
...
 <div class="infoBlock">
+ 
+ Follow our guide on <a href="https://rudderstack.com/docs/connections/adding-source-and-destination-rudderstack/">Adding a Source and Destination in RudderStack</a> for more information.
+</div>
 
-Please follow our guide on <a href="https://rudderstack.com/docs/connections/adding-source-and-destination-rudderstack/">How to Add a Source and Destination in RudderStack</a> to add a source and destination in RudderStack.
+- Assign a name to the destination and click on **Next**. You should then see the following screen:
 
-</div>
+<img src="../../assets/event-stream-destinations/customerio-connection-settings.png" alt="Customer.io connection settings" />
```
Comment (saikumarrs) [inline destinations/marketing/customer.io.mdx:43]: Can you please update this image as well?

Comment (ameypv-rudder) [reply destinations/marketing/customer.io.mdx:43]: this is now updated - apologies, I forgot to update the placeholder image.

**Review by ameypv-rudder — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**